examples/cc26xx/cc26xx-web-demo/project-conf.h

```diff
@@ -41,6 +41,13 @@
 #define CC26XX_WEB_DEMO_CONF_6LBR_CLIENT      1
 #define CC26XX_WEB_DEMO_CONF_COAP_SERVER      1
 #define CC26XX_WEB_DEMO_CONF_NET_UART         1
+#define CC26XX_WEB_DEMO_CONF_ADC_DEMO         1
+
+/*
+ * ADC sensor functionality. To test this, an external voltage source should be
+ * connected to DIO23
+ */
+#define CC26XX_WEB_DEMO_ADC_DEMO              0
```

Comment:
This should be changed to `CC26XX_WEB_DEMO_CONF_ADC_DEMO` sorry about that. @chenek do you want me to change it or will you do it?